public/index.html

```diff
@@ -5,6 +5,8 @@
   <title>Mozfest 2015</title>
   <link rel="stylesheet" href="index.css"/>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicon/favicon.png" />
```

Comment:
might as well add the 180x180 icon as normal icon:
```
<link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicon/favicon.png">
<link rel="icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="assets/images/favicon/apple-touch-icon-180x180.png">
```

Also note that `<link>` elements are void elements, and so don't need that `/>` at the end (just like `<img>`, `<meta>`, `<br>`)